UserGuide.md

```diff
@@ -260,6 +261,36 @@ gson.fromJson(json, fooType);
 ```
 The idiom used to get `fooType` actually defines an anonymous local inner class containing a method `getType()` that returns the fully parameterized type.
 
+### <a name="TOC-Parsing-a-Collection-with-Objects-of-Non-Arbitrary-Types"></a>Parsing a Collection with Objects of Non Arbitrary Types
+
+At times you will try to parse a collection of a specific object. The `getJson` and `fromJson`
```

Comment:
There is no `getJson` method (neither on `Gson` nor on any other class), which method do you mean?